packages/components/pie-toast/src/toast.scss

```
@@ -25,6 +27,43 @@
   line-height: var(--toast-line-height);
 }
 
+.c-toast--variant-info {
```

So to be consistent with what we currently use elsewhere variant classes would need to be:

```suggestion
.c-toast--info {

// or

.c-toast--variantInfo {
```